src/main/java/com/nbe8101team03/domain/user/entity/User.java

```diff
@@ -1,20 +1,17 @@
-package domain.member.entity;
+package com.nbe8101team03.domain.user.entity;
 
 import jakarta.persistence.*;
-import lombok.AccessLevel;
-import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Entity
 @Getter
-@Table(name="member")
+@Table(name="USERS")
```

Comment:
테이블 이름을 소문자로 통일하는게 어떨까요?
```suggestion
@Table(name="users")
```